src/scheduler/workers/analytics/addAnalyticsAuditEvent.ts

```diff
@@ -0,0 +1,40 @@
+import { prisma } from "scheduler/workers/model";
+
+const truncateMessage = "... truncated due to max length";
+const MAX_BYTES = 7000 - Buffer.byteLength(truncateMessage);
+
+export async function addAnalyticsAuditEvent(view: string, dataSent: any[], response: any) {
```

Comment:
yep, old habit 🙂. I don't think it actually makes a difference in this case since I never cast - it's only there to be stringified and logged but using never is a better shout over all.